src/screens/ContactScreen/index.js

```diff
@@ -262,9 +280,8 @@ const styles = StyleSheet.create({
   containerStyle: {
     marginHorizontal: 16
   },
-  subtitleStyle: (selectedUsers) => ({
-    color: selectedUsers.length > 0 ? COLORS.anon_primary : COLORS.gray4,
-    marginEnd: 8
+  subtitleStyle: (sourceScreen) => ({
+    color: sourceScreen === ANONYMOUS ? COLORS.anon_primary : COLORS.signed_primary
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The subtitle style now depends on whether the source screen is `ANONYMOUS` or not. If it is, the color is set to `COLORS.anon_primary`; otherwise, it's set to `COLORS.signed_primary`. Ensure that these color constants are defined and provide a good user experience in terms of contrast and readability.
```diff
+ subtitleStyle: (sourceScreen) => ({
+   color: sourceScreen === ANONYMOUS ? COLORS.anon_primary : COLORS.signed_primary
+ })
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->